platform/darwin/src/MGLLineStyleLayer.h

```diff
@@ -0,0 +1,100 @@
+// This file is generated. 
+// Edit platform/ios/scripts/generate-style-code.js, then run `make style-code-ios`.
+
+#import <Mapbox/Mapbox.h>
+
+#import "MGLTypes.h"
+
+typedef NS_ENUM(NSUInteger, MGLLineStyleLayerLineCap) {
```

Comment:
We’ll need documentation comments for enums and their values. Otherwise, they won’t show up in jazzy. We could either write the comments manually, as overrides, or we could have the comment simply say `@see -[MGL…Layer lineCap]`.